doc/0_quick_start.md

```diff
@@ -35,59 +35,65 @@ This driver does not depend on the Cassandra server code.
 ### Building
 See [Building And Artifacts](12_building_and_artifacts.md)
 
-### Preparing example Cassandra schema
-Create a simple keyspace and table in Cassandra. Run the following statements in `cqlsh`:
-
-```sql
-CREATE KEYSPACE test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1 };
-CREATE TABLE test.kv(key text PRIMARY KEY, value int);
-```
-      
-Then insert some example data:
-
-```sql
-INSERT INTO test.kv(key, value) VALUES ('key1', 1);
-INSERT INTO test.kv(key, value) VALUES ('key2', 2);
-```
- 
-Now you're ready to write your first Spark program using Cassandra.
-
 ### Loading up the Spark-Shell
 
-Run the `spark-shell` with the packages line for your version. To configure
+Run the `spark-shell` with the packages line for your version. This will include the connector
+and *all* of its dependencies on the Spark Class PathTo configure
 the default Spark Configuration pass key value pairs with `--conf`
 
     $SPARK_HOME/bin/spark-shell --conf spark.cassandra.connection.host=127.0.0.1 \
                                 --packages com.datastax.spark:spark-cassandra-connector_2.11:2.5.0
```

Comment:
This should now be an artifact with the "_2.12" name now, right?

Reply:
Ah, nevermind, I missed the 2.5.0 part... thanks for the clarification @RussellSpitzer